db/migrate/20200607025146_add_ancestry_to_vm.rb

```diff
@@ -0,0 +1,38 @@
+class AddAncestryToVm < ActiveRecord::Migration[5.2]
+  include RelationshipAncestryConverter
```

Comment:
this should be loaded via [spec_helper]
Probably want to scope this

```suggestion
  include ::RelationshipAncestryConverter
```

[spec_helper]: https://github.com/ManageIQ/manageiq-schema/blob/d30ccc15125a7c19b23182ba46bd96a05a12e9fb/spec/spec_helper.rb#L14

if that doesn't work, wonder if using extend `instead` would do what you need - but 
include` seems to be the right thing to me right now

Reply:
thank you  👍